examples/interacting_with_js_arrays/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+#[macro_use]
+extern crate neon;
+
+use neon::vm::{Call, JsResult};
+use neon::js::{Variant, JsNumber, JsArray};
+
+// https://github.com/dfrankland/node-eyeliner/blob/master/native/src/js_object_utils.rs
+pub fn js_value_to_float(js_number: Variant) -> f64 {
+  match js_number {
+    Variant::Number(number) => number.value(),
+    _ => panic!("heck"),
+  }
+}
+
+fn accepts_js_arrays(call: Call) -> JsResult<JsNumber> {
+  let js_arr_handle = call.arguments.get(call.scope, 0).unwrap();
+  let js_arr = js_arr_handle.downcast::<JsArray>().unwrap().to_vec(call.scope)?;
+
+  let vec_of_numbers: Vec<_> = js_arr.iter().map(move|js_value| {
+    let js_number = js_value.variant();
+    js_value_to_float(js_number)
+  }).collect();
+  let sum = vec_of_numbers.iter().sum();
+  Ok(JsNumber::new(call.scope, sum))
+}
+
+register_module!(m, {
+  try!(m.export("accepts_js_arrays", accepts_js_arrays));
+	Ok(())
+});
+
+// in lib/index.js
```

Comment:
Is there a reason to write this comment here, instead of just having that code in the JS file?

Reply:
I originally did it this way because I wasn't sure what structure would work in the examples and was trying to keep it minimal. I moved everything to mirror the output of `neon new` since I think that makes more sense.